suse_caasp

```diff
@@ -378,24 +382,76 @@ if check_rpm kubernetes-client && [ -f /etc/kubernetes/admin.conf ]; then
     log_cmd $OF 'kubectl version'
     log_cmd $OF 'kubectl api-versions'
     log_cmd $OF 'kubectl config view'
+    log_cmd $OF 'kubectl -n kube-system get pods --field-selector=status.phase=Running'
```

Comment:
Why do we want to retrieve the `Running` ones only? I think it would be valuable to have as much information as possible and wouldn't filter others out. WDYT?

Reply:
Good idea. Done